### PR TITLE
Cp 191 select existing member

### PR DESCRIPTION
**Pull request by RihabRJAB**

### CODE REVIEW

File: cockpit-frontend/src/components/TeamMemberList/TeamMemberList.jsx [hunk 1]
```diff
@@ -186,16 +196,35 @@ export default function TeamMemberList() {
...
-              <Button
-                className={classes.addButton}
-                variant="outlined"
-                color="primary"
-                startIcon={<AddIcon />}
-                onClick={displayForm}
-              >
-                Add A Member
-              </Button>
+              <Grid container spacing={1}>
```
Comment (zuming-acn) [inline cockpit-frontend/src/components/TeamMemberList/TeamMemberList.jsx:199]: In this script line 172, 2 lines are commented, is it necessary to keep them ?

{/*  <span className={classes.emailText}>{member.email}</span>
                      <br /> */}

File: cockpit_api/src/main/java/com/cockpit/api/controller/TeamController.java [hunk 2]
```diff
@@ -51,8 +51,19 @@ public ResponseEntity createTeam(@PathVariable Long mvpId,
...
+            List<TeamDTO> teamList = teamService.findAll();
+            return ResponseEntity.ok(teamList);
+        } else {
+            return ResponseEntity.ok(HttpStatus.UNAUTHORIZED);
+        }
+    }
+
+    // GET ALL MEMBERS
+    @GetMapping(value = "/api/v1/teamMember/all")
+    public ResponseEntity getAllMembers(@RequestHeader("Authorization") String authHeader) {
```
Comment (zuming-acn) [inline cockpit_api/src/main/java/com/cockpit/api/controller/TeamController.java:63]: ResponseEntity<Object> in order to remove sonar code smell

File: cockpit_api/src/main/java/com/cockpit/api/controller/TeamController.java [hunk 3]
```diff
@@ -140,11 +168,12 @@ public ResponseEntity deleteTeamMember(@PathVariable("id") Long id,
             return ResponseEntity.ok(HttpStatus.UNAUTHORIZED);
         }
     }
+
     // UNASSIGN a Team member
     @DeleteMapping(value = "/api/v1/team/{id}/unassignTeamMember/{teamMemberId}")
     public ResponseEntity<Object> unassignTeamMember(@PathVariable("id") Long id,
-                                           @PathVariable("teamMemberId") Long teamMemberId,
-                                           @RequestHeader("Authorization") String authHeader) {
+                                                     @PathVariable("teamMemberId") Long teamMemberId,
```
Comment (zuming-acn) [inline cockpit_api/src/main/java/com/cockpit/api/controller/TeamController.java:175]: It comes to me that unassign is an action of "update", but we created it as a DeleteMapping, how about modifying it to @PutMapping(should modify the unit test, too) ?

File: cockpit_api/src/main/java/com/cockpit/api/controller/TeamController.java (hunk 2 above)
Comment (zuming-acn) [reply cockpit_api/src/main/java/com/cockpit/api/controller/TeamController.java:63]: For all old controllers, this action has been done in the branch of code smell fixing

File: cockpit_api/src/main/java/com/cockpit/api/controller/TeamController.java (hunk 3 above)
Comment (RihabRJAB) [reply cockpit_api/src/main/java/com/cockpit/api/controller/TeamController.java:175]: done

File: cockpit_api/src/main/java/com/cockpit/api/controller/TeamController.java (hunk 2 above)
Comment (RihabRJAB) [reply cockpit_api/src/main/java/com/cockpit/api/controller/TeamController.java]: done

File: cockpit-frontend/src/components/TeamMemberList/TeamMemberList.jsx (hunk 1 above)
Comment (RihabRJAB) [reply cockpit-frontend/src/components/TeamMemberList/TeamMemberList.jsx:197]: done

**Review by zuming-acn — APPROVED** [review]

**Pull request merged**